GOVERNANCE.md

```diff
@@ -0,0 +1,47 @@
+<!-- This is heavily adapted version of
+the Benevolent dictator governance model by Ross
+Gardler and Gabriel Hanganu licensed under a Creative Commons 
+Attribution-ShareAlike 4.0 International License. -->
+
+# Overview
+This project is led by a benevolent dictator and mangaged by a core team of developers and a large community of contributors and users. That is, the community and core developers actively contribute to the day-to-day maintenance of the project, but the general strategic line is drawn by the benevolent dictator. In case of disagreement, they have the last word.
+
+# Roles And Responsibilities
+## Benevolent dictator (Hadley Wickham, @hadley)
+The job of the benevolent dictator is to set the strategic objectives of the project and communicate these clearly to the community, ensuring that the project survives in the long term. 
+
+## [Core Developers](https://github.com/orgs/tidyverse/teams/ggplot2)
+Core developers are contributors who have made several valuable contributions to the project and are now relied upon to both write code directly to the repository and screen the contributions of others. Committers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project. Many core developers are also [package authors](https://ggplot2.tidyverse.org/authors.html).
```

Comment:
I think as a matter of policy, all core developers (even if no longer active) should be included in authors.

Reply:
And maybe mention that they're empowered to merge PRs (after review)?